### PR TITLE
Feat(content) : rep changes with mereti and sestor

### PR DESCRIPTION
**Pull request by Hurleveur**
**Feature:** This PR implements the content follow up from PR #6351
related issue : [#5566](https://github.com/endless-sky/endless-sky/issues/5566)

## Feature Details
Kor Mereti and Kor Sestor now have their crew equivalent attribute. This means that them and other factions can mind you attacking them. However, nobody cares enough about them (not worse than -0.01) for that to matter, I changed the navy(oathkeepers) to have that by default. 
This means that before you do the wanderer campaign, nobody care about you fighting the Sestor/Mereti except for the faction you are killing. When the Mereti are pacified though this means that attacking them and doing any boarding/destroying/capturing will result in permanent reputation loss, meaning you can be at war with them.
Then, when the Kor Sestor drones are driven off to human space, I made the Navy(OathKeepers) have a -.1 relation with them, it was -.5 before but it meant getting 2k rep from saving the oathkeepers, which felt like it was too much. Now it will only be 400 rep from those missions, and you can keep on attacking kor sestor afterwards but farming pirates is more efficient.

## UI Screenshots
N/A

## Usage Examples
No more donut farming without consequence, they will remember you for what you are.

## Testing Done
I tried destroying or just attacking Kor Mereti/Sestor fleets, nobody gave a damn, except the Mereti themselves post pacification when you attack them.
[Kor Merethi~PACIFICATION.txt](https://github.com/endless-sky/endless-sky/files/7692925/Kor.Merethi.PACIFICATION.txt)
The Navy(OathKeepers) gives a damn from the moment they actually get assaulted by the sestor.
[Kor Merethi~SESTOR GO WILD.txt](https://github.com/endless-sky/endless-sky/files/7692923/Kor.Merethi.SESTOR.GO.WILD.txt)

## Performance Impact
N/A

### CODE REVIEW

**Comment by Hurleveur** [comment]
I saw someone mentioning the drak needing a crew equivalent attribute as well, should I do it in this PR as well ?
and what value should it be ? high enough that it means war with the quarg but friendship with the pug to kill them ? like 100 ? or just small to make sure that once you kill one the other archons are hostile ?

**Comment by Zitchas** [comment]
I'd go with about 100, that seems reasonable.

**Comment by Hurleveur** [comment]
just noticed : nobody cares about the Drak more than 1% (Pug, Quarg) (except for amber waste for which it actually should be 0.01) so it would not change any reps with anyone other than the Drak to kill them, if I just give them a crew equivalent attribute
should I change that too while I'm at it and tune it up to 0.1 ? I feel like they would mind it...

but all of that doesn't really matter actually because most drak are spawned by a mission that exists from the start of your game so it would not have much effect on the old saves (because their exact stats are already scripted in your save file)
idk how to make a patch for that... at least new saves would be good

File: data/governments.txt [hunk 1]
```diff
@@ -320,6 +320,14 @@ government "Kor Mereti"
 		"Kor Sestor" -.01
 		"Wanderer" -.01
 	
+	"penalty for"
+		assist 0
+		disable 0
+		board 0.5
+		capture 5
+		destroy 5
+		atrocity 0
```
Comment (Amazinite) [inline data/governments.txt:329]: Why the changes to the penalties? And why a penalty of 5 for destroying or capturing? You might as well just keep the capture and destroy penalties at their default and have the crew equivalent values be 5 times what they are now.

File: data/wanderer/wanderers middle.txt [hunk 2]
```diff
@@ -1747,6 +1747,9 @@ event "wanderers: kor sestor are aimless"
 		"required reputation" 1000
 	"reputation: Pirate" <?= -1
 	"unvisit planet" "Varu K'prai"
+	government "Navy (Oathkeeper)"
+		"attitude toward"
+			"Kor Sestor" -.1
```
Comment (Amazinite) [inline data/wanderer/wanderers middle.txt:1752]: I don't see a need for this, or at the very least the attitude should be reset back to -.01 after the Navy is done dealing with the Sestor, but I'd rather just not change the attitude toward to begin with.

**Review by Amazinite — CHANGES_REQUESTED** [review]

File: data/governments.txt (hunk 1 above)
Comment (Amazinite) [reply data/governments.txt:329]: These penalties should actually start at 0 then be changed to the default when the Mereti become friendly.

File: data/wanderer/wanderers middle.txt (hunk 2 above)
Comment (Hurleveur) [reply data/wanderer/wanderers middle.txt:1752]: It only makes sense that they like you more as you save them from annihilation by the hands of alien bots!
The thing is after you clear bots out of human space there are very little of them remaining, like you shut them down a few missions afterwards. 
I could make the reputation reset afterwards but they would appreciate you killing of the alien menace, even far away, at this point.

File: data/governments.txt (hunk 1 above)
Comment (Hurleveur) [reply data/governments.txt:329]: > These penalties should actually probably start at 0 then be changed to the default when the Mereti become friendly. The Sestor could also have their penalties be 0.

The thing is then we have to work with events which I don't really like, it doesn't hurt to have these penalties always be 5,, considering nobody cares about them enough for it to change your reputation, you just get lower rep with Mereti and Sestor which doesn't matter at all.
This way when smbd adds a way to make Sestor friendly for example, they won't have to create an event for it too. Also the Wanderer campaign could have the Mereti be hacked once again and turning back against you, it seems to me the simplest option just is to have it be this way and not touch it.

File: data/governments.txt [hunk 3]
```diff
@@ -320,6 +320,14 @@ government "Kor Mereti"
...
+
```
Comment (Amazinite) [inline data/governments.txt:330]:
```suggestion
```

File: data/wanderer/wanderers middle.txt (hunk 2 above)
Comment (Amazinite) [inline data/wanderer/wanderers middle.txt:1752]:
```suggestion
```

**Review by Amazinite — CHANGES_REQUESTED** [review]
